Library/Source/Console.cpp

```diff
@@ -38,6 +40,7 @@ namespace babylon
             OutputDebugStringA(info[index].ToString().Utf8Value().c_str());
         }
         OutputDebugStringA("\r\n");
+#endif
```

Comment:
nit: Else cout? Or stderr? Just so the message goes somewhere. I think standard outputs end up in logcat on Android (though that might be _very_ out-of-date information).

Reply:
You are right, Logcat is not outdated info :). I'm taking a look now at how to organize Android platform dependent stuff.